src/plugins/data_views/server/fetcher/index.ts

```diff
@@ -8,7 +8,8 @@
 
 export * from './index_patterns_fetcher';
 export {
-  shouldReadFieldFromDocValues,
-  mergeCapabilitiesWithFields,
   getCapabilitiesForRollupIndices,
+  mergeCapabilitiesWithFields,
+  shouldReadFieldFromDocValues,
 } from './lib';
+export type { RollupIndexCapability } from './lib';
```

Comment:
This needs to be exported since its part of a public API

Reply:
Could you document it? Trying to prevent `Public APIs missing comments` from increasing